packages/odyssey-storybook/src/components/odyssey-mui/Table/Table.stories.tsx

```diff
@@ -0,0 +1,472 @@
+/*!
+ * Copyright (c) 2021-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import { Story } from "@storybook/react";
+import * as React from "react";
+import Button from "@mui/material/Button";
+import Table from "@mui/material/Table";
+import TableBody from "@mui/material/TableBody";
+import TableCell from "@mui/material/TableCell";
+import TableContainer from "@mui/material/TableContainer";
+import TableHead from "@mui/material/TableHead";
+import TableRow from "@mui/material/TableRow";
+import TableSortLabel from "@mui/material/TableSortLabel";
+import Checkbox from "@mui/material/Checkbox";
+import { MuiThemeDecorator } from "../../../../.storybook/components";
+import TableMdx from "./Table.mdx";
+
+export default {
+  title: `MUI Components/Table`,
+  component: Table,
+  parameters: {
+    docs: {
+      page: TableMdx,
+    },
+  },
+  argTypes: {
+    disabled: {
+      control: "boolean",
+      defaultValue: false,
+    },
+    icon: {
+      control: "text",
+      defaultValue: null,
+    },
+    label: {
+      control: "text",
+      defaultValue: "Asteroids",
+    },
+    wrapped: {
+      control: "boolean",
+      defaultValue: false,
+    },
+  },
+  decorators: [MuiThemeDecorator],
+};
+
+interface Data {
+  name: string;
+  radius: number;
+  type: string;
+  perihelion: string;
+  descriptor: string;
+}
+
+function createData(
+  name: string,
+  radius: number,
+  type: string,
+  perihelion: string,
+  descriptor: string
+): Data {
+  return {
+    name,
+    radius,
+    type,
+    perihelion,
+    descriptor,
+  };
+}
+
+const rows = [
+  createData("Mercury", 2440, "Terrestrial", "--", "Mercurian"),
+  createData("Venus", 6051, "Terrestrial", "--", "Venusian"),
+  createData("Earth", 6378, "Terrestrial", "January 4, 2023", "Terran"),
+  createData("Mars", 3396, "Terrestrial", "June 21, 2022", "Martian"),
+  createData("Jupiter", 71492, "Gas giant", "January 21, 2023", "Jovian"),
+  createData("Saturn", 60268, "Gas giant", "November 29, 2032", "Saturnian"),
+  createData("Uranus", 25559, "Ice giant", "August 17-19, 2050", "Uranian"),
+  createData("Neptune", 24764, "Ice giant", "September 4, 2042", "Neptunian"),
+];
```

Comment:
Not fond of not using named props like are in the MUI docs.

Looking at this data, I can't tell what `2440` is at a glance.

Reply:
The types for each are declared above this dataset. The current setup is a quick solve to get the data into the `Table` while we work on a permanent solution.